src/SDK/SmartDetectors.Extensions/AlertExtensions.cs

```diff
@@ -594,6 +591,16 @@ private static ContractsThresholdType ConvertThresholdTypeToContractsThresholdTy
 
         #endregion
 
+        private static string CombinePropertyNames(string parentPropertyName, string propertyName)
```

Comment:
> CombineP [](start = 29, length = 9)

CreatePropertyName?
ConstructPropertyName?

Reply:
I think "combine" is better - same as Path.Combine

---
In reply to: [262425951](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/56#discussion_r262425951) [](ancestors = 262425951)